### ISSUE TITLE
Add environment file

### ISSUE THREAD

**Issue by evasterviga** [Vtrana/VS_project1#1]
I cannot run your script. I get the following error:
```
Traceback (most recent call last):
  File "merging_data.py", line 4, in <module>
    from bioinfokit import analys, visuz
ModuleNotFoundError: No module named 'bioinfokit'
```

Basically my local environment does not contain all the packages you import in your code. To help the user out with this you can create an environment file where you include the requirements for running your code. Then I as the user can run this file to create the necessary environment. To find out how to do this, have a look [here](https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/04-sharing-environments/index.html) for instance.

I saw that you have written requirements in the README.md file. However, you have written hw to import them in th epython script. But I would like you to create an environment file and write how to activate that environment OR to write what commands I need to paste in my command line to install the packages.

**Comment by Vtrana** [comment]
Dear Emma,
I have now uploaded environment file. I do not know if it was created in a proper way and will be sufficient.
That is something new for me.

Hope it will work.
Best regards
Valentina

From: EmmaCAndersson ***@***.***>
Sent: Monday, October 24, 2022 11:19 AM
To: Vtrana/VS_project1 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Vtrana/VS_project1] Add environment file (Issue #1)


I cannot run your script. I get the following error:

Traceback (most recent call last):

  File "merging_data.py", line 4, in <module>

    from bioinfokit import analys, visuz

ModuleNotFoundError: No module named 'bioinfokit'

Basically my local environment does not contain all the packages you import in your code. To help the user out with this you can create an environment file where you include the requirements for running your code. Then I as the user can run this file to create the necessary environment. To find out how to do this, have a look here<https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/04-sharing-environments/index.html> for instance.

**Comment by evasterviga** [comment]
Hi,

The environments file looks ok. But you will need to change some things to make it work. 
If I try to create the environment I cannot install pandas and bioinfokit. You can solve this by adding **channels** to the environment.yml file (check out the link I sent you on creating environment files).  For pandas you can use conda-forge for example. You can see [here](https://anaconda.org/conda-forge/pandas) that to install pandas using conda you can use conda-forge. For bioinfokit you can use bioconda.

Please try to use it to create an environment to check that it works. You can do that like this:
```
conda env create -n ENVIRONMENT_NAME --file environment.yml
```

Then, if you want to delete the environment later:

```
conda env remove -n ENVIRONMENT_NAME 
```

**Comment by Vtrana** [comment]
dear Emma,
Thank you for your guidance,
I hope it is fixed now, can create virt environment and run it in Anaconda. 
Best regards
Valentina

**Comment by evasterviga** [comment]
Yes, now it works! Nice explanatory additions to the README too!
